include/ppx/knob.h

```diff
@@ -314,6 +316,67 @@ class KnobDropdown final
     std::vector<T> mChoices;
 };
 
+// KnobConstant is intended for parameters that cannot be adjusted when the application is run
+// They will not be displayed in the UI and cannot have their values changed by the application
+template <typename T>
+class KnobConstant final
+    : public Knob
+{
+public:
+    KnobConstant(const std::string& flagName, T defaultValue)
+        : Knob(flagName, false)
+    {
+        SetValue(defaultValue);
+    }
+
+    KnobConstant(const std::string& flagName, T defaultValue, T minValue, T maxValue)
```

Comment:
Just wondering do we need this constructor, or could we just use the constructor above and add a validator function?